<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="admin-PC" ignored="0" name="Test(failed)" tests="3" failures="3" timestamp="24 Aug 2021 14:17:26 GMT" time="0.045" errors="0">
  <testcase name="tc3" time="0.011" classname="org.testngnew.RerunExamples">
    <failure type="java.lang.AssertionError" message="Check Tc3 Again expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Check Tc3 Again expected [true] but found [false]
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
  <testcase name="tc2" time="0.001" classname="org.testngnew.RerunExamples">
    <failure type="java.lang.AssertionError" message="Check Tc2 Again expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Check Tc2 Again expected [true] but found [false]
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
  <testcase name="tc1" time="0.001" classname="org.testngnew.RerunExamples">
    <failure type="java.lang.AssertionError" message="Check tc1 again expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Check tc1 again expected [true] but found [false]
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc1 -->
</testsuite> <!-- Test(failed) -->
